api-operator/pkg/registry/utils/utils.go

```diff
@@ -59,3 +61,24 @@ func IsImageExists(auth RegAuth, imageRepository string, imageName string, tag s
 	}
 	return false, nil
 }
+
+func GetPullSecrets(pullSecretDef string, pullSecretName string) []corev1.LocalObjectReference {
+	// var pullSecretDef = &PullSecretDefined
+	// var pushSecretDef = &registry.PushSecretDefned
+	var pullSecretArray = []corev1.LocalObjectReference{}
```

Comment:
Shall we remove this commented code?

Reply:
Removed. Please review

Reply:
Fixed via [f1b3ac5](https://github.com/wso2/k8s-api-operator/pull/555/commits/f1b3ac5d7418d9479b19828218edb9cb7a8b7001)